README.rst

```diff
@@ -31,58 +31,64 @@ Installation
 
 .. code-block:: console
 
-    (<custom virtualenv>)$ pip install invenio-cli
+    $ pip install invenio-cli
 
 Usage
 =====
 
-
 Local Development environment
 -----------------------------
 
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    (<custom virtualenv>)$ invenio-cli init --flavour=RDM
-    (<custom virtualenv>)$ cd <created folder>
+    $ invenio-cli init --flavour=RDM
+    $ cd <created folder>
 
-    # Install python dependencies, link/copy assets + statics, install js
+    # Install locally
+    # install python dependencies, link/copy assets + statics, install js
     # dependencies, build assets and final statics
-    (<custom virtualenv>)$ invenio-cli build --local --pre
+    $ invenio-cli install
 
     # Start and setup services (database, Elasticsearch, Redis, queue)
-    (<custom virtualenv>)$ invenio-cli services --local
+    $ invenio-cli services
 
     # Optional: add demo data
-    (<custom virtualenv>)$ invenio-cli demo --local
+    $ invenio-cli demo --local
```

Comment:
Hope: I don'r really see how, but is there a way or getting rid of the `--local`/`--containers` for the demo command? We might even not consider it? and just say "If you want to use it you can do so through `invenio` (rdm-records)"?

Reply:
On the question of having `--local/--containers`, It might be the case that running `invenio-cli demo` (without any option) would be enough to cover both cases, since they could potentially use the same DB container/volume. 

On the question of having `invenio-cli demo` at all,  it is quite convenient for the marketing case of: "just run these commands and you will see search results and records on your local installation". If we remove it, we have to tell people to run `docker exec <name of your UI container> /bin/bash -c "invenio demo"` which might be much just to fullfill the marketing case.

Reply:
> Marketing case

Agree :)

> demo command without option.

Yup, I hope. Is just we come back to the issue of where is `invenio-rdm-records` installed the venv or the container :( I was just wondering if you now see a way of solving this with the new CLI :)

(Feel free to shelf it)

Reply:
Added it as a point on https://github.com/inveniosoftware/invenio-cli/issues/108